api/content/api.raml

```diff
@@ -0,0 +1,93 @@
+#%RAML 1.0 ResourceType
+title: Libero content API
+
+uses:
+  base: ../../core/api.raml
+
+types:
+    ID:
+        type: string
+        pattern: ^[A-Za-z0-9\-._]+$
+    Version:
+        type: LatestVersion | NumberVersion
+    LatestVersion:
+        type: string
+        enum: [latest]
+    NumberVersion:
+        type: integer
+        minimum: 1
+
+/{id}:
+    type:
+        base.item:
+            resource: Content item
+    description: |
+        Single content item.
+    uriParameters:
+        id:
+            description: |
+                Content item ID.
+            type: ID
+
+    delete:
+        description: |
+            Delete a content item.
+        responses:
+            202:
+                description: |
+                    Deletion request accepted.
+
+    /versions:
+        description: |
+            Versions of a content item.
+
+        post:
+            description: |
+                Create a content item version.
+            responses:
+                202:
+                    description: |
+                        Creation request accepted.
```

Comment:
Is there a sample of the body of this? For https://github.com/libero/libero/issues/32

Reply:
Not yet, will start now. Basically going to be each possible part in a wrapper (eg one `<front>` per language).

Reply:
Is there a JSON sequence equivalent in the XML world? (Multiple documents together without a container.) Would mean avoiding having another schema.

Reply:
Unless it's a very simple container (allows anything to be included, then individually validated).

Reply:
Started in 14e19bdc8c527a07b72c25942d4aa3ecdf52f049.